inspec/a2-api-smoke/controls/base.rb

```diff
@@ -50,10 +50,8 @@
     },
   },
   'teams-service': {
-    path: '/api/v0/auth/teams/version',
-    response_match: {
-      name: 'teams-service'
-    },
+    path: '/apis/iam/v2/teams',
```

Comment:
Maybe mark this as a "todo"; my PR re-introduces the version endpoint for teams v2, so then this original test would work.

Reply:
which PR is that? could you let me know if it merges before this one?

Reply:
issue #2928

Reply:
Yours is ready to merge, right? Mine is still in the works.